test-case/verify-kernel-boot-log.sh

```diff
@@ -115,7 +115,10 @@ dmic_switch_present()
     (set -x
      # name= is hardcoded in /usr/share/alsa/ucm2/*
      # This returns a non-zero error status on failure
-     amixer cget name='Dmic0 Capture Switch'
+     #amixer cget name='Dmic0 Capture Switch'
```

Comment:
Why keep the old code commented out? git accident?

The comment on line 117 is now in the wrong place, it must follow the `amixer` command.